snapcraft/common.py

```diff
@@ -98,20 +103,36 @@ def __init__(self):
         super().__init__(
             '{0} is not supported, please log a bug at '
             'https://bugs.launchpad.net/snapcraft/+filebug?'
-            'field.title=please+add+support+for+{0}'.format(
-                platform.machine()))
+            'field.title=please+add+support+for+{0}'.format(_machine))
```

Comment:
I know you didn't add the title in this PR, but I wanted to voice a small concern. We're introducing a dependency here on Launchpad's URL format. Do we know that will remain stable? What happens in a few years when this code is still used but Launchpad renames `title` to be `summary` for example? We'll still be printing this but people will get a 404.